examples/Example_embed_shift_template_experiment.ipynb

```diff
@@ -0,0 +1,865 @@
+{
```

Comment:
This is a great point! I think that I'll try to add more detail on how the data is processed so that users can make a more informed decision. Basically the template and compendium need to be in the same "scale" because the same scaling factor used to normalize the compendium is applied to the new template experiment. So if the template has a very different range of values, then you will have outliers after normalization that the VAE cross entropy loss won't work.